azure/template.json

```
@@ -30,6 +30,12 @@
         "certificateName": {
             "type": "string"
         },
+        "sharedApimServiceName": {
```

```suggestion
        "sharedApimName": {
```
I've added a new SharedApimName variable to the variable groups